pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -71,8 +71,9 @@ class HilbertSchmidt(Operation):
 
         .. code-block:: python
 
-            with qml.tape.QuantumTape(do_queue=False) as u_tape:
-                qml.Hadamard(wires=0)
+            with qml.tape.QuantumTape() as u_tape:
+                with qml.QueuingManager.stop_recording():
```

Comment:
I think we need to flip the order, no?
```suggestion
            with qml.QueuingManager.stop_recording():
                with qml.tape.QuantumTape() as u_tape:
```
This way, `Hadamard` will be in `u_tape`, but `u_tape` will not be queued in an outer context.

Reply:
Oh, nice catch!